dvc.yaml

```diff
@@ -26,7 +26,6 @@ plots:
       y: digit
       template: bar_horizontal
       title: Histogram of Predictions
-  - training/plots/images
```

Comment:
Any reason this was dropped?

Reply:
mistake

Reply:
fixed here: https://github.com/iterative/vscode-dvc-demo/pull/42

Reply:
@dberenbaum please LMK (when you have time) if there are any further changes that need to get made to remove legacy syntax 🙏🏻